src/assets/thirdPartyServices/mozi/gene_annotation_service/form/index.js

```diff
@@ -200,7 +200,11 @@ const AnnotationForm = props => {
     <div className="container form-wrapper">
       {/* Gene List */}
       <Typography variant="h6" gutterBottom>
-        Input Genes
+        Input{" "}
+        <a href="http://www.genenames.org" target="_blank">
```

Comment:
you have to use `rel="noopener"` , whenever you're using `target="_blank"`..
instead use the `AnchorLink` component from our common directory `src/components/common/AnchorLink.js`. 
you just have to pass `newTab` as a prop, if you want to open it in new tab, it will take care of the rest.